examples/secret_channel.py

```diff
@@ -0,0 +1,106 @@
+# import discord and any other libraries
+import discord
+from discord.ext import commands
+from discord.utils import get
+
+# bot description
+description = "A simple example showcasing how to make text and voice channels (as well as private ones)."
+
+# create a bot instance with prefix "?" and description
+bot = commands.Bot(command_prefix="?", description=description)
+
+@bot.event
+async def on_ready():
+    """Prints information to the console once the bot is ready."""
+
+    print('Logged in as')
+    print(bot.user.name)
+    print(bot.user.id)
+    print('Bot Online!')
+
+def can_create():
+    """A check to see if the user has the required manage_channels permission."""
+    
+    async def predicate(ctx):
+        member = ctx.author
+        return member.guild_permissions.manage_channels
+    return commands.check(predicate)
+
+@bot.group(invoke_without_command=True)
+@can_create()
+async def create(ctx, *, new_channel: str):
```

Comment:
The signature of a command is user-facing as well. This means that when someone runs a help command on it they can see the signature and be pointed in the right direction. In this case the parameter should be `channel_name` not `new_channel`, since the command name already implies that it's creating something.

As an aside, I feel like the command name doesn't really make much sense on its own but I guess for an example there can be leeway here.